ppx_test/test.ml

```diff
@@ -229,9 +229,10 @@ let test_default_enum () =
   check_unmarshal_error enum_of_rpc (Rpc.Int 1L);
   check_unmarshal_error enum_of_rpc (Rpc.Float 1.0)
 
-type enum_string_map = (enum * string) list [@@deriving rpc]
+type enum2 = [`a | `b | `c] [@@deriving rpc]
+type enum_string_map = (enum2 * string) list [@dict] [@@deriving rpc]
```

Comment:
Please keep both tests: the one using `@dict` and the one not using it.

Reply:
Good point